app/models/container.rb

```diff
@@ -2,6 +2,7 @@ class Container < ApplicationRecord
   include SupportsFeatureMixin
   include NewWithTypeStiMixin
   include ArchivedMixin
```

Comment:
@zeari Aah, I was referring the https://github.com/ManageIQ/manageiq/pull/14322/files#diff-bfb9598d272bb5d013105604b37dcabaR3

Ok, now that I look at the code, please remove it from `ArchivedPurgingMixin`. Do you think a better name would be `PurgingContinerMixin`?